docs/release-notes/november-2023.md

```diff
@@ -24,12 +24,13 @@ tags:
 ## 🔧 Semgrep OSS Engine
 
 :::note
-Starting from version 1.46.0, Semgrep is first released in the following ecosystems:
+Beginning with version 1.46.0, Semgrep is first released to:
+
 - `pypy`
 - `brew`
 - `returntocorp/semgrep:canary` (Docker)
 
-If no issues are detected after a few days, the Semgrep team then promotes the `:canary` Docker tag to `:latest`.
+If no issues are detected after a few days, Semgrep promotes the `:canary` Docker tag to `:latest`.
```

Comment:
Let's differentiate what Semgrep (the product) does from what the maintainers of Semgrep actually do. This is not an automatic process - the PA oncall is monitoring this actively.